src/Modal.jsx

```diff
@@ -73,6 +73,7 @@ class Modal extends React.Component {
 
     this._entering = this._entering.bind(this)
     this._exiting  = this._exiting.bind(this)
+    this._removeUnknownDivProps = this._removeUnknownDivProps.bind(this)
```

Comment:
no need to bind the the method, it's not passed around and doesn't reference 'this'